configs/default/IFRC-JBF7.config

```diff
@@ -0,0 +1,212 @@
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin A01 0
+# pin A01: DMA1 Stream 6 Channel 3
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature OSD
+feature ESC_SENSOR
+
+# beacon
+beacon RX_LOST
+beacon RX_SET
+
+# serial
+serial 0 64 115200 57600 0 115200
+serial 3 1024 115200 57600 0 115200
+serial 4 2048 115200 57600 0 115200
+
+# vtxtable
+vtxtable bands 1
+vtxtable channels 8
+vtxtable band 1 RACEBAND R FACTORY    0    0 5732 5769 5806 5843    0    0
+vtxtable powerlevels 1
+vtxtable powerlabels 25
+
+# master
+set gyro_to_use = BOTH
+set mag_bustype = I2C
+set mag_i2c_device = 2
+set mag_hardware = NONE
+set baro_bustype = I2C
+set baro_i2c_device = 2
+set baro_hardware = NONE
+set serialrx_provider = SBUS
+set adc_device = 3
+set blackbox_p_ratio = 64
+set blackbox_device = SDCARD
+set dshot_burst = ON
+set motor_pwm_protocol = DSHOT600
+set vbat_max_cell_voltage = 440
+set current_meter = ESC
+set battery_meter = ADC
+set vbat_scale = 100
+set ibata_scale = 100
+set beeper_inversion = ON
+set beeper_od = OFF
+set sdcard_mode = SPI
+set sdcard_spi_bus = 3
+set osd_warn_batt_not_full = OFF
+set osd_warn_batt_warning = OFF
+set osd_warn_batt_critical = OFF
+set osd_warn_esc_fail = OFF
+set osd_warn_core_temp = OFF
+set osd_rssi_alarm = 40
+set osd_cap_alarm = 5000
+set osd_vbat_pos = 384
+set osd_rssi_pos = 2426
+set osd_tim_1_pos = 53
+set osd_tim_2_pos = 2455
+set osd_flymode_pos = 2420
+set osd_throttle_pos = 2400
+set osd_vtx_channel_pos = 34
+set osd_crosshairs_pos = 236
+set osd_ah_pos = 199
+set osd_current_pos = 2440
+set osd_mah_drawn_pos = 2448
+set osd_craft_name_pos = 2048
+set osd_display_name_pos = 490
+set osd_gps_speed_pos = 271
+set osd_gps_lon_pos = 82
+set osd_gps_lat_pos = 65
+set osd_gps_sats_pos = 1410
+set osd_home_dir_pos = 302
+set osd_home_dist_pos = 303
+set osd_compass_bar_pos = 265
+set osd_altitude_pos = 246
+set osd_pid_roll_pos = 135
+set osd_pid_pitch_pos = 167
+set osd_pid_yaw_pos = 199
+set osd_debug_pos = 0
+set osd_power_pos = 320
+set osd_pidrate_profile_pos = 344
+set osd_warnings_pos = 2406
+set osd_avg_cell_voltage_pos = 2432
+set osd_pit_ang_pos = 256
+set osd_rol_ang_pos = 288
+set osd_battery_usage_pos = 391
+set osd_disarmed_pos = 75
+set osd_nheading_pos = 310
+set osd_nvario_pos = 278
+set osd_esc_tmp_pos = 82
+set osd_esc_rpm_pos = 83
+set osd_stat_max_spd = OFF
+set max7456_spi_bus = 2
+set dashboard_i2c_bus = 2
+set pinio_config = 129,1,1,1
+set pinio_box = 0,255,255,255
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_2_spibus = 1
+set gyro_2_sensor_align = CW90
+set gyro_2_align_yaw = 900
+
+# rateprofile 0
+set roll_rc_rate = 127
+set pitch_rc_rate = 127
+set roll_expo = 40
+set pitch_expo = 40
+set roll_srate = 72
+set pitch_srate = 72
+set yaw_srate = 75
```

Comment:
Including rates in a target config is a bad idea. Besides being a personal preference setting, many people fly with default rates and this flight controller would behave differently since it has different default rates.